dattri/func/fisher.py

```diff
@@ -201,7 +201,21 @@ def _single_datainf_ifvp(
         grad: torch.Tensor,
         regularization: float,
     ) -> torch.Tensor:
-        # TODO: docstring
+        """Intermediate DataInf ifvp calculation of a single training data point.
```

Comment:
I think we could get rid of ifvp_datainf now?

Reply:
Do you mean completely delete the implementations of ifvp_datainf and at_x versions

Reply:
Got it!

Reply:
The current unit test involves test of ifvps as well and I can remove them after the test is updated?

Reply:
I think you could update the test in this PR as well.

Reply:
Sure. Removing all tests of ifvp_datainf